src/hip_stream.cpp

```diff
@@ -31,12 +31,21 @@ THE SOFTWARE.
 //-------------------------------------------------------------------------------------------------
 // Stream
 //
+#if (__hcc_minor__ < 3)
```

Comment:
HIP/VDI is compiled with gcc. This condition will always be true.

Should it be #if __HCC__ && (__hcc_minor__ < 3) ?